src/collections/_documentation/clients/cocoa/index.md

```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
 
 We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
+### SPM | Swift Package Manage
+
+Starting with sentry-cocoa version `4.4.3`, we support Swift Package Manager.
+
+To integrate Sentry into your xCode project using SPM, open your App in xCode and open File -> Swift Packages -> Add Package Dependency. Then add sentry-cocoa by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or brach) on the next page.
+
+NOTE: Version tags or branches need to have the Package.swift file in it or xCode won't be able to install the package. Thus versions previous to `4.4.3` can't be installed via SPM!
```

Comment:
Does this doc consistently use “xCode” versus “Xcode?”

Reply:
thanks